rialto_airflow/mais.py

```diff
@@ -7,19 +6,11 @@
 
 import requests
 from requests_oauthlib import OAuth2Session
-import dotenv
-
-# Load environment variables
-dotenv.load_dotenv()
 
 # Type aliases for clarity
 ORCIDRecord = dict[str, Any]
 ORCIDStats = list[Union[str, int, float]]
 
-BASE_URL = os.environ.get("AIRFLOW_VAR_MAIS_BASE_URL")
-CLIENT_ID = os.environ.get("AIRFLOW_VAR_MAIS_CLIENT_ID")
-CLIENT_SECRET = os.environ.get("AIRFLOW_VAR_MAIS_SECRET")
-
```

Comment:
we don't actually need to set any of these values here -- we are always passing them in from the dag when they are used